contracts/prize-strategy/PrizeStrategy.sol

```diff
@@ -384,6 +388,18 @@ contract PrizeStrategy is PrizeStrategyStorage,
     }
   }
 
+  /// @notice Removes an external ERC721 token as an additional prize that can be awarded
+  /// @dev Only the Prize-Strategy owner/creator can remove external tokens
+  /// @param _externalErc721 The address of an ERC721 token to be removed
+  /// @param _prevExternalErc721 The address of the previous ERC721 token in the list.
+  /// If no previous, then pass the SENTINEL address: 0x0000000000000000000000000000000000000001
+  function removeExternalErc721Award(address _externalErc721, address _prevExternalErc721) external onlyOwner {
+    require(externalErc721s.contains(_externalErc721), "PrizeStrategy/invalid-external-award");
```

Comment:
Same thoughts here... I think we've already covered by:

`require(self.addressMap[prevAddress] == addr, "Invalid prevAddress");`